stable/sonatype-nexus/README.md

```diff
@@ -63,6 +63,8 @@ The following table lists the configurable parameters of the Nexus chart and the
 | Parameter                                   | Description                         | Default                                 |
 | ------------------------------------------  | ----------------------------------  | ----------------------------------------|
 | `replicaCount`                              | Number of Nexus service replicas    | `1`                                     |
+| `deploymentStrategy`                        | Deployment Strategy     |  `rollingUpdate` |
+| `namespace`                        | pvc namespace     |  `default` |
```

Comment:
Not sure I understand the use case of manually setting the pvc namespace. Helm charts generally control resource within the same namespace.